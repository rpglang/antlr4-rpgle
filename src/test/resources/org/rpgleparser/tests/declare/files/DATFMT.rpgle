       dcl-f table1 DATFMT(*YMD/);
       dcl-f table2 DATFMT(*JUL&);
       dcl-f table3 DATFMT(*YMD-);