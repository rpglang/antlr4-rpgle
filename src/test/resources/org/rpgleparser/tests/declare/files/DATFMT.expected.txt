/*===TOKENS===*/
FS_FreeFile              <dcl-f>
ID                       <table1>
KEYWORD_DATFMT           <DATFMT>
'('                      <(>
SPLAT_YMD                <*YMD/>
')'                      <)>
FREE_SEMI                <;>
FS_FreeFile              <dcl-f>
ID                       <table2>
KEYWORD_DATFMT           <DATFMT>
'('                      <(>
SPLAT_JUL                <*JUL&>
')'                      <)>
FREE_SEMI                <;>
FS_FreeFile              <dcl-f>
ID                       <table3>
KEYWORD_DATFMT           <DATFMT>
'('                      <(>
SPLAT_YMD                <*YMD->
')'                      <)>
FREE_SEMI                <;>
/*===TREE===*/
(r
  (statement
    (fspec
      dcl-f
      (filename table1)
      (fs_keyword
        (keyword_datfmt
          DATFMT
          (
          (simpleExpression (identifier (all (symbolicConstants *YMD/))))
          )
        )
      )
      ;
    )
  )
  (statement
    (fspec
      dcl-f
      (filename table2)
      (fs_keyword
        (keyword_datfmt
          DATFMT
          (
          (simpleExpression (identifier (all (symbolicConstants *JUL&))))
          )
        )
      )
      ;
    )
  )
  (statement
    (fspec
      dcl-f
      (filename table3)
      (fs_keyword
        (keyword_datfmt
          DATFMT
          (
          (simpleExpression (identifier (all (symbolicConstants *YMD-))))
          )
        )
      )
      ;
    )
  )
)
/*======*/